querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -284,6 +265,151 @@ export const DataDocScheduleForm: React.FunctionComponent<
     );
 };
 
+const NotifactionFormRow: React.FC<{
+    name: string;
+    index: number;
+    removeRow: (index: number) => void;
+    notificationRow: IDataDocScheduleNotification;
```

Comment:
these values don't need to be passed, since you are already calling `useFormikContext`

Reply:
not using `useFormikContext` anymore